addons/web/static/src/model/relational_model/static_list.js

```diff
@@ -365,6 +365,13 @@ export class StaticList extends DataPoint {
         });
     }
 
+    archive(record) {
```

Comment:
why don't you instead call record.update({ active: false }) on the subrecord? I would prefer that don't do any change in the relational_model...